packages/core/database/src/index.ts

```diff
@@ -58,6 +58,8 @@ class Database {
 
   entityManager: EntityManager;
 
+  readonly DEFAULT_MAX_IDENTIFIER_LENGTH = 55 as const;
```

Comment:
Did we switch from 64 to 55?

Reply:
63 is the hard maximum for postgres, we went with 55 to reserve a buffer in case we need to add table prefixes, for example (like for environments, multitenancy, etc)